src/main/java/com/nha/abdm/wrapper/hrp/CommonHelpers/CareContextBuilder.java

```diff
@@ -1,4 +1,4 @@
-package com.nha.abdm.wrapper.hrp.common;
+package com.nha.abdm.wrapper.hrp.CommonHelpers;
```

Comment:
Since these are models, can we move these to the `model` package?

Reply:
The name of the class can be just `CareContext`

Reply:
Renamed the CommonHelpers package to model and also renamed CareContext